1-js/05-data-types/01-primitives-methods/article.md

```diff
@@ -28,30 +28,30 @@ let john = {
 john.sayHi(); // Hi buddy!
 ```
 
-So, here we've made an object `john` with the method `sayHi`.
+So here we've made an object `john` with the method `sayHi`.
 
-There exist many built-in objects, including those that work with dates, errors, HTML elements etc. They have different properties and methods.
+Many built-in objects already exist, such as those that work with dates, errors, HTML elements etc. They have different properties and methods.
 
-But features come at a price!
+But, these features come with a cost!
 
-Objects are "heavier" than primitives. They require additional resources to support the internal machinery. But properties and methods are useful in programming, JavaScript engines try to optimize them, so the price is usually fair.
+Objects are "heavier" than primitives. They require additional resources to support the internal machinery. But as properties and methods are very useful in programming, JavaScript engines try to optimize them to reduce the additional burdon.
 
 ## A primitive as an object
 
 Here's the paradox faced by the creator of JavaScript:
 
-- There are many things one would want to do with a primitive like a string or a number. Could be great to access them as methods.
+- There are many things one would want to do with a primitive like a string or a number. It would be great to access them as methods.
 - Primitives must be as fast and lightweight as possible.
 
-The solution looks a little bit awkward, but here it is.
+The solution looks a little bit awkward, but here it is:
 
 1. Primitives are still primitive. A single value, as desired.
-2. The language allows to access methods and properties of strings, numbers, booleans and symbols.
-3. When it happens, a special "object wrapper" is created that provides the functionality and then is destroyed.
+2. The language allows access to methods and properties of strings, numbers, booleans and symbols.
+3. When this happens, a special "object wrapper" is created that provides the extra functionality, and then is destroyed.
```

Comment:
Why "extra", why not just "the functionality" ?

Reply:
Equivalent, but a bit more descriptive.

Reply:
ok